components/fern/test.vue

```diff
@@ -9,7 +9,7 @@
                         Test with
                         <span
                             class="text-gradient font-semibold from-violet-500 to-sky-400"
-                            >confidence</span
+                            >Confidence</span
```

Comment:
I wasn't sure which pieces of text belonged in title-case vs sentence-case capitalization. This seems more consistent with the top half of the homepage, whereas lowercase would be more consistent with the lower half. Either way, probably good to pick one standard and run with it everywhere.